standard/expressions.md

```diff
@@ -4378,16 +4380,16 @@ The tuple equality operator `x != y` is evaluated as follows:
 
 ### 12.12.12 The is operator
 
-There are two forms of the `is` operator.  One is the *is-type operator*, which has a type on the right-hand-side.  The other is the *is-pattern operator*, which has a pattern on the right-hand-side.
+There are two forms of the `is` operator. One is the *is-type operator*, which has a type on the right-hand-side. The other is the *is-pattern operator*, which has a pattern on the right-hand-side.
 
 #### 12.12.12.1 The is-type operator
 
 The *is-type operator* is used to check if the run-time type of an object is compatible with a given type. The check is performed at runtime. The result of the operation `E is T`, where `E` is an expression and `T` is a type other than `dynamic`, is a Boolean value indicating whether `E` is non-null and can successfully be converted to type `T` by a reference conversion, a boxing conversion, an unboxing conversion, a wrapping conversion, or an unwrapping conversion.
 
 The operation is evaluated as follows:
 
-1. If `E` is an anonymous function, a compile-time error occurs
-1. If `E` is a method group or the `null` literal, of if the value of `E` is `null`, the result is `false`.
+1. If `E` is an anonymous function, lambda expression, or method group, a compile-time error occurs
```

Comment:
I thought so too. I'll leave this until our meeting because one of the comments was that this bullet "didn't cover lambda expressions". I don't know if that was a stale comment, or if one of had something different in mind.